### PR TITLE
implement a stop button for background tasks

### PR DESCRIPTION
**Pull request by lorenzejay**
stopbackgroundtaskbutton

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,6 +70,12 @@
         "command": "runme.resetEnv",
         "category": "Runme",
         "title": "Reset Environment Variables for current VS Code session"
+      },
+      {
+        "command": "runme.stopBackgroundTask",
+        "category": "Runme",
+        "title": "Dispose terminal session to stop background task."
+
```
Comment (christian-bromann) [inline package.json:78]: Let's add this to below section `commandPalette` to avoid having this command shown there.

**Review by christian-bromann — COMMENTED** [review]
Can we have a unit test for `StopBackgroundTaskProvider` and `stopBackgroundTask`?

File: package.json (hunk 1 above)
Comment (lorenzejay) [reply package.json:78]: will do.

**Comment by lorenzejay** [comment]
@christian-bromann whenever you got time, can you review.

**Comment by christian-bromann** [comment]
Is this a duplicate of #50 ?

**Comment by lorenzejay** [comment]
> Is this a duplicate of #50 ?

#50 worked on top of this branch, so in a way yes, this would be redundant.

**Comment by christian-bromann** [comment]
@lorenzejay I see, I switched the base branch in #50 to be on top of this one for better clarity.

**Review by christian-bromann — APPROVED** [review]
👍

**Comment by lorenzejay** [comment]
@christian-bromann can this be merged. Merged conflicts resolved

**Comment by sourishkrout** [comment]
Any chance we can shorted the `label` and use `$(debug-stop)` or an icon that makes more sene from https://code.visualstudio.com/api/references/icons-in-labels?

**Comment by lorenzejay** [comment]
> Any chance we can shorted the `label` and use `$(debug-stop)` or an icon that makes more sene from https://code.visualstudio.com/api/references/icons-in-labels?
> 
> cc @christian-bromann @lorenzejay

will do !

**Comment by lorenzejay** [comment]
@sourishkrout  @christian-bromann  2 options `task vs command.`


Screenshot 2022-11-16 at 12 43 29 PM
Screenshot 2022-11-16 at 12 41 54 PM

**Comment by lorenzejay** [comment]
this stop icon might be better ?
Screenshot 2022-11-16 at 12 46 56 PM

**Comment by sourishkrout** [comment]
The icon works... can rename it to `Stop Task` please? It's shorter and goes along with `Background Task`

**Comment by christian-bromann** [comment]
Closing in favor of https://github.com/stateful/vscode-runme/pull/63

**Pull request closed without merging**